src/Evolution/DiscontinuousGalerkin/Limiters/Minmod.hpp

```diff
@@ -122,20 +122,22 @@ namespace Limiters {
 /// the characteristic variables requires specializing the limiter to each
 /// evolution system.
 ///
-/// The limiter acts in the `Frame::Logical` coordinates, because in these
-/// coordinates it is straightforward to formulate the algorithm. This means the
-/// limiter can operate on generic deformed grids --- however, some things can
-/// start to break down, especially on strongly deformed grids:
-/// 1. When the Jacobian (from `Frame::Logical` to `Frame::Inertial`) varies
+/// The limiter acts in the `Frame::ElementLogical` coordinates, because in
+/// these coordinates it is straightforward to formulate the algorithm. This
+/// means the limiter can operate on generic deformed grids --- however, some
+/// things can start to break down, especially on strongly deformed grids:
+/// 1. When the Jacobian (from `Frame::ElementLogical` to `Frame::Inertial`)
+/// varies
```

Comment:
Missing two spaces may break rendering (haven't checked). Perhaps fix in a follow-up and merge this quickly.